dependencies.toml

```diff
@@ -1,67 +1,148 @@
 [versions]
 akka = "2.6.19"
+akka-http-cors = "1.0.0"
```

Comment:
Double checked whether there were any mistakes in the migration and I can confirm that all versions are correctly migrated to the `[versions]` table.

Reply:
Thanks for the double check 😄